chainercv/links/model/segnet/segnet_basic.py

```diff
@@ -133,23 +138,23 @@ def __call__(self, x):
             An image-wise score. Its channel size is :obj:`self.n_class`.
 
         """
-        p1 = F.MaxPooling2D(2, 2, use_cudnn=False)
-        p2 = F.MaxPooling2D(2, 2, use_cudnn=False)
-        p3 = F.MaxPooling2D(2, 2, use_cudnn=False)
-        p4 = F.MaxPooling2D(2, 2, use_cudnn=False)
+        p1 = F.MaxPooling2D(2, 2)
+        p2 = F.MaxPooling2D(2, 2)
+        p3 = F.MaxPooling2D(2, 2)
+        p4 = F.MaxPooling2D(2, 2)
         h = F.local_response_normalization(x, 5, 1, 1e-4 / 5., 0.75)
-        h = p1(F.relu(self.conv1_bn(self.conv1(h), test=not self.train)))
-        h = p2(F.relu(self.conv2_bn(self.conv2(h), test=not self.train)))
-        h = p3(F.relu(self.conv3_bn(self.conv3(h), test=not self.train)))
-        h = p4(F.relu(self.conv4_bn(self.conv4(h), test=not self.train)))
+        h = _without_cudnn(p1, F.relu(self.conv1_bn(self.conv1(h))))
```

Comment:
How about.
I am not sure what is the best practice to do this, but I think the code below looks easier to read.

```python
with chainer.using_config('use_cudnn', 'never'):
    h = p1(F.relu(....))
    h = p2(F.relu(....))
    h = p3(F.relu(....))
    h = p4(F.relu(....))

```